os/fs/mqueue/mq_close.c

```diff
@@ -155,7 +155,7 @@ int mq_close_group(mqd_t mqdes, FAR struct task_group_s *group)
 
 int mq_close(mqd_t mqdes)
 {
-	FAR struct tcb_s *rtcb = (FAR struct tcb_s *)sched_self();
+	FAR struct tcb_s *rtcb;
```

Comment:
We don't need to get this after sched_lock. Let's remove line 167 instead of deleting here.

Reply:
@jeongarmy You didn't follow my suggestion. Please let me know your opinion.